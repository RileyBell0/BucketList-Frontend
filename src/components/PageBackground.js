import "../styles/pageBackground.css";

// Draws the background image and color behind all other elements (z-index -998)
function PageBackground() {
  return (
    <>
      <div className="page-background__common page-background__color" />
      <div className="page-background__common page-background__image" />
    </>
  );
}

export default PageBackground;
