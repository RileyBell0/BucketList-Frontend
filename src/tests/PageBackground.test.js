import renderer from "react-test-renderer";
import PageBackground from "../components/PageBackground";

it("Matches previous snapshot", () => {
  const comp = renderer.create(<PageBackground />);
  let tree = comp.toJSON();
  expect(tree).toMatchSnapshot();
});
