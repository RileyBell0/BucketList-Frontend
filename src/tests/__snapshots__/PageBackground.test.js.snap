// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches previous snapshot 1`] = `
Array [
  <div
    className="page-background__common page-background__color"
  />,
  <div
    className="page-background__common page-background__image"
  />,
]
`;
